sw/host/rom_ext_image_signer/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "rom_ext_signer"
+version = "0.1.0"
+authors = ["lowRISC contributors"]
+edition = "2018"
+
+[profile.dev]
+panic = "abort"
+lto = false
+opt-level = "z"
+debug = true
+
+[profile.release]
+panic = "abort"
+lto = true
+opt-level = "z" # TODO - opt-level = 3 ?
+debug = true
+
+[dependencies]
+# TODO - do we want a fixed version, minor number updates or the latest?
```

Comment:
I strongly suggest pinning to an exact version, because it avoids having to trust upstreams from doing the right thing (I don't have very much faith in the ecosystem not making mistakes).

Reply:
+1 to pinning to an exact version to avoid surprises.